src/app/clusters/color-control-server/color-control-server.cpp

```diff
@@ -809,9 +818,14 @@ bool ColorControlServer::moveHueCommand(app::CommandHandler * commandObj, const
 
     // New command.  Need to stop any active transitions.
     stopAllColorTransitions(endpoint);
+    // now, kick off the state machine.
+    initHueTransitionState(endpoint, colorHueTransitionState, isEnhanced);
 
     if (moveMode == EMBER_ZCL_HUE_MOVE_MODE_STOP)
     {
+        // Per spec any staturation transition must also be cancelled.
```

Comment:
Do you believe this should not be the case? 

To me, This action on receipt mentions that the MoveMode stop can be used to stop other Move commands

```
Table 17. Actions on Receipt for MoveSaturation Command

MoveMode | Action on Receipt -- | --
Stop |  If moving, stop, else ignore the command (i.e., the command is accepted but has no effect). NB This MAY also be used to stop a MoveToSaturation command, a MoveToHue command, or a MoveToHueAndSaturation command.
```

Reply:
That has no SHALL-level normative requirements that I can see.

Reply:
here is the spec issue
https://github.com/CHIP-Specifications/connectedhomeip-spec/issues/6328